ulas/chain.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "ULAS",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "ULAS",
+  "chain_id": "ulas",
+  "bech32_prefix": "ulas",
```

Comment:
Please register this prefix to SLIP 173
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Done